### ISSUE TITLE
Design Badges

### ISSUE THREAD

**Issue by kristinashu** [mozilla/teach.mozilla.org#1242]
We still don't know what the exact badges will be but I'd like to start thinking about their overall look.

Previous badges can be seen here: https://blog.webmaker.org/badges
these adhere to MoCo's badge guidelines http://emilygoligoski.com/2014/05/07/introducing-mozilla-visual-templates-for-badge-issuers/
I think these where kept simple so they could be generated through a web app and anyone could make one and assign to someone.

All of our badges will be created by us so I'm wondering if you think we need to follow these badge guidelines or if we could branch out? I think we could do something much more exciting - more detailed illustrations, maybe different shaped badges, incorporate titles, maybe make them gifs with very slight animation...

Here are a few nice examples:
https://dribbble.com/shots/1924150-Ain-t-No-Mountain-High-Enough-Badge
https://dribbble.com/shots/991400-Montreal-Meets-2013
https://dribbble.com/shots/2227935-Osprey-Packs-Patch
https://dribbble.com/shots/2063687-Leaders-Academy
https://dribbble.com/shots/2144627-Patch-Badge-3

@hannahkane @cassiemc any thoughts?

**Comment by hannahkane** [comment]
This feels like largely a design question, so I'll defer to @cassiemc. I will say that my one flag would be that I wouldn't want badge design to get so complicated that it's a blocker for spinning up a new badge when we need one. On the other hand, those examples on dribbble are super cute.

**Comment by cassiemc** [comment]
I love the dribbble examples and agree we can push the design here. There are also some really lovely badge examples about mid-way through this case study: http://beta.rallyinteractive.com/national-parks/ 

A few questions though:

* What's happening to all the old badges? If we updated the system now, would we need to update the old badges? The idea of an incoherent set makes me feel yucky.
* How important is it for non-designers to be able to spin up badges? That was one reason they were kept simple, not just for generative purposes. That said, more often than not it was the designers in the end who were engaged to put together the final badge designs.
* Where else is the badge system being used? My understanding was that this system was used well beyond Mofo-land, in various Moco places too. 
* I DO think that people's ability to get excited about badges will contribute to the success of badges. And that's super duper important. However, we always had SO much to do. So the badge designs have historically taken lower priority. This doesn't concern me as much given I think we have the right people in place now to move quickly, but I did want to flag it for @hannahkane's sake.

@kristinashu All in all, I am FOR this update of the design. It would be way better, and more inline with the evolution of our brand/s. I'd like to understand the complexities but am still keen to do it. 

Even moving away from font-awesome would be great, I find those icons super clunky, and if we have icon-ability in house there's really no reason why we shouldn't. And that really might lend itself to subtle animations later on, if not now... which is an idea I absolutely adore.

**Comment by hannahkane** [comment]
Answers to your questions:
 * We won't be promoting old Mofo badges, so I don't think consistency is an issue there.
 * I don't think we need non-designers to be able to create badge designs, but, ideally, spinning up a new badge should take no more than an hour or two of a designer's time. I'd had to lose days each time we want to create a new badge.
 * The badge system we're designing is for MLN only. Moz-wide, badges are used elsewhere, and I don't think anything we do would prevent them from continuing to use the old badge design system if they want.
 * Agreed that nice looking badges are a win for user engagement. Just want to make sure we're on the same page about what "move quickly" means. What do you think is a reasonable amount of time for a designer to invest in a badge design?

**Comment by anmechung** [comment]
+1 to Hannah's responses, and was about send out a calendar invite to Kristina to brainstorm designs for the 21C Skill badges.

**Comment by kristinashu** [comment]
Sounds good. @cassiemc - I really like the parks case study.

I'll work on designs which are not constrained by the existing badge designs but are also not too complicated.

@anmechung let me know when you're free to brainstorm.

**Comment by cassiemc** [comment]
I'd say 2-3 hours is a reasonable amount of time for a custom badge if we have a system / style aesthetic in place and well documented. Someone who hasn't designed a badge before might take another hour or two to ramp up and become familiar with the new aesthetic. I'd allot 2-3 days toward completing a new visual system, though. Does that sound reasonable @kristinashu? 

Seems worthwhile to me, but your call @hannahkane!

**Comment by hannahkane** [comment]
+1 to investing in a badge design system!

**Comment by kristinashu** [comment]
Time estimates seem good to me. Exploring different design options at this initial stage will be the hard/fun part but once a system is in place it should go quickly.

**Comment by LauraReynal** [comment]
cc :shipit:

**Comment by kristinashu** [comment]
Update from call with @anmechung and Iris about 21st Century Skill badges:
https://etherpad.mozilla.org/21CSkillsbadgesdesign
An-Me and Iris will talk further about simplifying the number of badges and we'll talk again later on this week.

**Comment by kristinashu** [comment]
Here are few badge directions https://redpen.io/p/yzc4eaa85d7b6c16c7
My preference is for the second row of badges - they're more like the original badges than some of the other versions.

I've been testing out some icons from the Futuro Icons set https://www.iconfinder.com/bloomua
If we're open to purchasing icons, I think it would be a much quicker way to design all the badges.

Colours are from the Mozilla Unified palette.

@cassiemc can you please let me know if I'm heading in the right direction and if you have any other feedback?

**Comment by kristinashu** [comment]
@cassiemc I've added an updated JPG based on the feedback from our design crit  https://redpen.io/p/yzc4eaa85d7b6c16c7 Let me know what feedback you have.

**Comment by anmechung** [comment]
@kristinashu I've updated the number and types of badges:
https://etherpad.mozilla.org/21CSkillsbadgesdesign
will look at your calendar for time to connect on Friday.

**Comment by davidascher** [comment]
Nice!

2c:

* In general, sticking w/ the original design language (hexagons, m tab, etc.) is a win, as revisiting that system would require a bunch more stakeholders.  So, agree w/ preference for rows 1 & 2 .

* I'd like to see some explorations of how these badges would look in different contexts.  In particular, apart from a profile page, I expect many instantiations of badges to be much smaller resolution.  Doing multi-resolution iconography is v. hard, and readability/distinguishability of the icons ends up being a chalenge.

**Comment by anmechung** [comment]
@kristinashu is this the most recent version to comment on? https://redpen.io/p/tq1fbf0a41edd3909f

**Comment by kristinashu** [comment]
I've updated the badges to align more with the old badges and to ensure scalability https://redpen.io/p/tq1fbf0a41edd3909f
- darkened the stroke
- simplified the stitching
- increased size of the m tab
- added a slight radial gradient
- increased size of icons

I've also added a few example posts to give a better sense of context and scalability.

**Comment by hannahkane** [comment]
My two cents: I love these, and think they look great. Will leave it to other designers to offer more detailed feedback, and will invite @anmechung and @chrislarry33 to weigh in as well.

**Comment by anmechung** [comment]
Looking good!!   I would like the badges grouped in 4 different colors: Collaboration badges all one color; Innovation, Problem Solving, and Critical Thinking different color each. The goal being that it signifies that the collaboration badges are a set.  Thinking ahead about web literacy badges too.  Adding @IBGMozilla for comments.  Will also run these by those who will pilot these badges, and their young people too.

**Comment by IBGMozilla** [comment]
I agree. Really love how it's coming together. The diff colors will help to signify that the collaboration badges are grouped together.

**Comment by kristinashu** [comment]
@anmechung @IBGMozilla I did at one time have the badges grouped in different colors but it felt too colorful (especially with the achieved/pending/eligible tags). With so many colors, the colors started to loose their significance. I'll brainstorm to see if there's another way to distinguish the sub groupings.

@hannahkane @cassiemc do you think we should buy the icons? They're $2 each at https://www.iconfinder.com/bloomua Do you think this style is definitely the direction we're heading in? Once we buy the icons then I can start fine-tuning them.

**Comment by hannahkane** [comment]
@chrislarry33 - can you approve budget for a set of $2 icons we'll use for badges?

**Comment by chrislarry33** [comment]
@hannahkane @kristinashu Approved

**Comment by chrislarry33** [comment]
@anmechung will know which budget btw Mott/Irvine they get charged to

**Comment by kristinashu** [comment]
Comps are here https://redpen.io/p/tq1fbf0a41edd3909f

Closing this ticket and opening #1323 to add co-branding.
